specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -0,0 +1,292 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+@doc("The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.")
+@service({
+    title: "Microsoft Cognitive Language Service",
+    version: "2022-05-01",
+})
+namespace Azure.AI.Language.Conversations;
+
+@doc("Error Response")
+model ErrorResponse {
+    @doc("The error object.")
+    error: Error;
+}
+
+@doc("The error object.")
+model Error extends Record<unknown> {
+    @doc("One of a server-defined set of error codes.")
+    code: ErrorCode;
+
+    @doc("A human-readable representation of the error.")
+    message: string;
+
+    @doc("The target of the error.")
+    target?: string;
+
+    @doc("An array of details about specific errors that led to this reported error.")
+    details?: Array<Error>;
+
+    @doc("An object containing more specific information than the current object about the error.")
+    innererror?: InnerErrorModel;
+}
+
+@doc("An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.")
+model InnerErrorModel extends Record<unknown> {
+    @doc("One of a server-defined set of error codes.")
+    code: InnerErrorCode;
+
+    @doc("Error message.")
+    message: string;
+
+    @doc("Error details.")
+    details?: Record<string>;
+
+    @doc("Error target.")
+    target?: string;
+
+    @doc("An object containing more specific information than the current object about the error.")
+    innererror?: InnerErrorModel;
+}
+
+@doc("Language of the text records. This is BCP-47 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default.")
+model Language {}
```

Comment:
I wonder if `alias Language = string` would be better here. As written, this is an [`object`](https://github.com/Azure/azure-rest-api-specs/pull/24376/files/3915adc42d96d97dce11e6fca8130c4e86e1dd85..47d0c06834e36e377603adeb8b7a7094cfa017f9#diff-9ce96076a5b956c41c76a3c8ee9d2a3a8d416f8d83b857440456dcf01665eb8eR1121).